app/serializers/provider_serializer.rb

```diff
@@ -13,17 +13,22 @@ def unique_id
 
   def name
     if Flipper.enabled?(:facilities_ppms_caresite_name)
-      case object.ProviderType
-      when /GroupPracticeOrAgency/i
-        object.CareSite
-      else
-        object.ProviderName
-      end
+      fipper_enabled_name
     else
-      object.ProviderName
+      object.ProviderName || object.Name
     end
   end
 
+  def fipper_enabled_name
+    possible_name = case object.ProviderType
+                    when /GroupPracticeOrAgency/i
+                      object.CareSite
+                    else
+                      object.ProviderName
+                    end
+    [possible_name, object.Name].find(&:present?)
```

Comment:
I find myself getting tripped up on this line - is this equivalent to this?

```ruby
possible_name || object.Name
```

Reply:
it is, `possible_name || object.Name` was causing an issue with codeclimate

Reply:
Gotcha, weird, but they're functionally equivelant, yes? Suppose I'm just asking for my own curiosity.

Reply:
Also weird that codeclimate seems to be OK with line 18: `object.ProviderName || object.Name`, but not here? 🤔